playgrounds/petclinic-maven-sb-2/src/main/resources/templates/pets/createOrUpdatePetForm.html

```
@@ -0,0 +1,34 @@
+    <span th:text="#{pet}">Pet</span>
+  </h2>
+  <form th:object="${pet}" class="form-horizontal" method="post">
+    <input type="hidden" name="id" th:value="*{id}" />
+    <div class="form-group has-feedback">
+      <div class="form-group">
+        <label class="col-sm-2 control-label" th:text="#{owner}">Owner</label>
+        <div class="col-sm-10">
+          <span th:text="${owner?.firstName + ' ' + owner?.lastName}" />
+        </div>
+      </div>
+      <input th:replace="~{fragments/inputField :: input ('Name', 'name', 'text')}" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)